src/containers/LivingWhitepaper/containers/PrincipalEntities/index.tsx

```diff
@@ -1,10 +1,595 @@
+          </p>
+          <p className="PrincipalEntities__section-paragraph PrincipalEntities__section-paragraph--my-8">
+            Validator that accepts requests from other nodes; after successful validation of each request, it creates a
+            new block which is first added to it's own blockchain and then forwarded to the confirmation validators
+          </p>
+          <Divider className="PrincipalEntities__section-divider" />
+          <p className="PrincipalEntities__section-paragraph PrincipalEntities__section-paragraph--my-8 PrincipalEntities__section-text--black">
+            Points
+          </p>
+          <p className="PrincipalEntities__section-paragraph PrincipalEntities__section-paragraph--my-8">
+            An amount of refillable units that are awarded in exchange for locking coins
+          </p>
+          <Divider className="PrincipalEntities__section-divider" />
+          <p className="PrincipalEntities__section-paragraph PrincipalEntities__section-paragraph--my-8 PrincipalEntities__section-text--black">
+            Registered Users
+          </p>
+          <p className="PrincipalEntities__section-paragraph PrincipalEntities__section-paragraph--my-8">
+            Registered Users Individual users of the network that have a username
+          </p>
+          <Divider className="PrincipalEntities__section-divider" />
+          <p className="PrincipalEntities__section-paragraph PrincipalEntities__section-paragraph--my-8 PrincipalEntities__section-text--black">
+            Treasury Board
+          </p>
+          <p className="PrincipalEntities__section-paragraph PrincipalEntities__section-paragraph--my-8">
+            Highest voted members of government who mint new coins
+          </p>
+          <Divider className="PrincipalEntities__section-divider" />
+          <p className="PrincipalEntities__section-paragraph PrincipalEntities__section-paragraph--my-8 PrincipalEntities__section-text--black">
+            Username
+          </p>
+          <p className="PrincipalEntities__section-paragraph PrincipalEntities__section-paragraph--my-8">
+            A globally unique network identifier linked to an account number
+          </p>
+          <Divider className="PrincipalEntities__section-divider" />
+          <p className="PrincipalEntities__section-paragraph PrincipalEntities__section-paragraph--my-8 PrincipalEntities__section-text--black">
+            Vote
+          </p>
+          <p className="PrincipalEntities__section-paragraph PrincipalEntities__section-paragraph--my-8">
+            Registered users choice for governor
+          </p>
+        </section>
+        <Divider className="PrincipalEntities__divider" />
+        <div className="PrincipalEntities__bottom-bar">
+          <div
+            className="PrincipalEntities__bottom-bar-content"
+            onClick={() => history.push('/developer/whitepaper/principal-events')}
+            role="button"
+            tabIndex={0}
```

Comment:
can we make this a component under `LivingWhitepaper/components`? so we can reuse in other pages. Can pass in the URL and text to be displayed as props

Reply:
let me create the component from my side